home/.chezmoiscripts/universal/run_onchange_after_32-gitomatic.sh.tmpl

```diff
@@ -0,0 +1,68 @@
+{{- if eq .host.distro.family "linux" -}}
+#!/usr/bin/env bash
+# @file git-o-matic Configuration
+# @brief Configures and starts service(s) on Linux systems to monitor Git repositories
+# @description
+#     git-o-matic is a tool to monitor git repositories and automatically pull/push changes. Multiple repositories can be
+#     monitored by running multiple instances of `gitomatic`. Username/Token and SSH Key based authentication are supported
+#
+#     If the `gitomatic` program is installed, this script uses the configuration stored in `home/dot_config/gitomatic/config.yml.tmpl`
+#     (that unpacks with Chezmoi to `~/.config/gitomatic/config.yml`) to configure and start systemd user service(s). One service monitors
+#     one repository. It is possible to configure:
+#     * The interval between checks
+#     * The Author name and Email to use if Pushing is enabled,
+#     * Enable/disable Pushing and Pulling and
+#     * The authentication method to use
+#
+#     ## Notes
+#     * If SSH Key based authentication is to be used, ensure that the Public key is configured in Git Provider and that the Private Key
+#       file exists in the system at the path specified in the configuration
+#     * If Username/Token is to be used, provide the `username` value in plaintext in the configuration file. The Token needs to be encrypted
+#       and stored following the Install Doctor method documented [here](https://install.doctor/docs/customization/secrets), and the name of
+#       file should be provided in the configuration file against the `passwordTokenFile` field.
+#
+#     ## Links
+#
+#     * [gitomatic GitHub repository](https://github.com/muesli/gitomatic/)
+#     * [gitomatic configuration](https://github.com/megabyte-labs/install.doctor/blob/master/home/dot_config/gitomatic/config.yml.tmpl)
+#     * [Secrets / Environment variables documentation](https://install.doctor/docs/customization/secrets)
+
+{{ includeTemplate "universal/profile" }}
+{{ includeTemplate "universal/logg" }}
+
+### Create Systemd services to run gitomatic
+if command -v gitomatic > /dev/null; then
+  {{ if and (stat (joinPath .chezmoi.sourceDir "dot_config" "gitomatic" "config.yml.tmpl")) (gt (len (include "dot_config/gitomatic/config.yml.tmpl" | fromYaml ).repositories) 0) }}
+    ### Create directory to store Systemd service definition files
+    mkdir -p "$HOME/.config/systemd/user/"
+    {{ range (include "dot_config/gitomatic/config.yml.tmpl" | fromYaml ).repositories }}
```

Comment:
Starting `gitomatic` processes is moved to this new file under `~/.local/bin`. This file is executed by the systemd service. Since there is one process per repository, the config.yml file has been removed.